app-emulation/docker-cli/docker-cli-20.10.8.ebuild

```diff
@@ -34,7 +34,6 @@ src_prepare() {
 src_compile() {
 	# Flatcar: override go version
```

Comment:
is this comment still relevant ?

Reply:
Yes, it refers to `go_export` configuring `PATH` and other environment variables in a way that `go` refers to a fixed go version, specified by COREOS_GO_VERSION. This decouples us from using the latest go version. We will soon add go1.17 to the tree.